src/lib.rs

```diff
@@ -412,23 +413,8 @@ impl RenderPass {
                 None => wgpu::Origin3d::ZERO,
             };
 
-            let alpha_srgb_pixels: Option<Vec<_>> = match &image_delta.image {
-                egui::ImageData::Color(_) => None,
-                egui::ImageData::Font(a) => Some(a.srgba_pixels(Some(1.0)).collect()),
-            };
-
             let image_data: &[u8] = match &image_delta.image {
```

Comment:
Isn't clippy barking here? I would have guessed that it wants you to use `if let egui::ImageData::Color(_) =...` instead here?

Reply:
Okay, checked in the IDE. Clippy is fine with this.